src/backend/joanie/core/utils/issuers.py

```diff
@@ -31,3 +32,36 @@ def generate_document(name: str, context: dict) -> bytes:
         font_config=FontConfiguration(),
     )
     return doc_html.write_pdf(stylesheets=[css], zoom=1)
+
+
+def get_contract_definition_placeholder_default_values():
+    """
+    Return a dictionary with default placeholder values for previewing a template
+    of contract definition.
+    """
+    return {
+        "contract": {
+            "body": _("Default Articles template"),
+            "title": _("Contract Definition"),
+        },
+        "course": {"name": _("Course Python with Django")},
+        "student": {
+            "name": "John Doe",
+            "address": {
+                "address": "1 Rue de L'Exemple",
+                "city": "Paris",
+                "country": "FR",
+                "first_name": "John Doe",
+                "last_name": "",
+                "id": "1234",
+                "is_main": True,
+                "postcode": "75000",
+                "title": "Office",
+            },
+        },
+        "organization": {
+            "logo": "cover.png",
+            "name": _("Just The Best Ever Online School"),
+            "signature": "signature_organization.png",
+        },
```

Comment:
`logo` and `signature` are base64 encoded in the context. We could assume to pass empty values as placeholder or a square grey image ?

```
data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAMAAAAoyzS7AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAylpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDkuMS1jMDAxIDc5LjE0NjI4OTk3NzcsIDIwMjMvMDYvMjUtMjM6NTc6MTQgICAgICAgICI+IDxyZGY6UkRGIHhtbG5zOnJkZj0iaHR0cDovL3d3dy53My5vcmcvMTk5OS8wMi8yMi1yZGYtc3ludGF4LW5zIyI+IDxyZGY6RGVzY3JpcHRpb24gcmRmOmFib3V0PSIiIHhtbG5zOnhtcD0iaHR0cDovL25zLmFkb2JlLmNvbS94YXAvMS4wLyIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bXA6Q3JlYXRvclRvb2w9IkFkb2JlIFBob3Rvc2hvcCAyNS4wIChNYWNpbnRvc2gpIiB4bXBNTTpJbnN0YW5jZUlEPSJ4bXAuaWlkOjI1NzhEMDAzNkI1OTExRUU5OTNGQUJGODRGMTMzRTRCIiB4bXBNTTpEb2N1bWVudElEPSJ4bXAuZGlkOjI1NzhEMDA0NkI1OTExRUU5OTNGQUJGODRGMTMzRTRCIj4gPHhtcE1NOkRlcml2ZWRGcm9tIHN0UmVmOmluc3RhbmNlSUQ9InhtcC5paWQ6MjU3OEQwMDE2QjU5MTFFRTk5M0ZBQkY4NEYxMzNFNEIiIHN0UmVmOmRvY3VtZW50SUQ9InhtcC5kaWQ6MjU3OEQwMDI2QjU5MTFFRTk5M0ZBQkY4NEYxMzNFNEIiLz4gPC9yZGY6RGVzY3JpcHRpb24+IDwvcmRmOlJERj4gPC94OnhtcG1ldGE+IDw/eHBhY2tldCBlbmQ9InIiPz6Z7QuCAAAABlBMVEXa2tIAAABZ2pBsAAAADElEQVR42mJgAAgwAAACAAFPbVnhAAAAAElFTkSuQmCC
```

Reply:
I think the grey squares sounds good. Do you have any recommendations to set the pixels of those gray squares please ?